cbcontainers/state/components/enforcer_deployment.go

```diff
@@ -32,8 +32,8 @@ var (
 
 	EnforcerAllowPrivilegeEscalation       = false
 	EnforcerReadOnlyRootFilesystem         = true
-	EnforcerRunAsUser                int64 = 0
-	EnforcerCapabilitiesToAdd              = []coreV1.Capability{"NET_BIND_SERVICE"}
```

Comment:
does this mean that if I override the port in the CRD it will fail, because of the missing capability? I wasn't aware you need this cap. to bind on 443

Reply:
Yes, if one overrides it with a port <1000, then the capability is required. 
Note that the service still exposes port 443 and calls should be going through the service as opposed to using the container port.